### PR TITLE
Only assign the first reviewer

### PR DESCRIPTION
**Pull request by wilzbach**
Ping @MartinNowak @klickverbot - I think only assigning one reviewer is better because it creates a clear responsibility

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -353,6 +353,8 @@ async function work(body) {
     });
   } else {
     createComment(data, message);
+    // for now we only pick the best matching reviewer
+    reviewers = [reviewers[0]];
     assignReviewer(data, reviewers);
```
Comment (MartinNowak) [inline server.js]: `reviewers.slice(0, 1)`

**Comment by dnadlinger** [comment]
Agreed.

**Comment by wilzbach** [comment]
> error: failed to push some refs to 'git@github.com:dlang-bots/mention-bot.git'

@MartinNowak why are you always faster? ;-)

**Comment by MartinNowak** [comment]
> @MartinNowak why are you always faster? ;-)

emacs ;)

**Pull request merged**